### PR TITLE
CCDF for New York

### PR DESCRIPTION
**Pull request by tolaouk**
Adding CCDF eligibility parameters and variables for New York

### CODE REVIEW

**Comment by MaxGhenis** [comment]
Ah actually the rates vary by county as well, per https://ocfs.ny.gov/main/policies/external/OCFS_2016/LCMs/16-OCFS-LCM-18.pdf#page=14

Though that report is from 2016, we should use a newer one.

These market rates aren't in the PRD, so we should also see how they do it.

File: openfisca_us/variables/finance/benefit/ccdf.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    entity = SPMUnit
+    definition_period = YEAR
+    documentation = ""
+
+    def formula(spm_unit, period, parameters):
+        # Get state and poverty ratio for SPM unit.
+        state = spm_unit("spm_unit_state", period)
+        assets = spm_unit("spm_unit_assets", period)
+        # Get parameters.
+        p_asset_limit = parameters(period).benefit.CCDF.eligibility.assets[
```
Comment (MaxGhenis) [inline openfisca_us/variables/finance/benefit/ccdf.py:18]:
```suggestion
        p_asset_limit = parameters(period).benefit.ccdf.eligibility.assets[
```
Once #229 is merged, all variables and parameters will be lowercase, so you'll need to update a few places here.

File: openfisca_us/parameters/benefit/CCDF/eligibility.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+description: CCDF eligibility by income and assets.
+
+initial_income:
```
Comment (MaxGhenis) [inline openfisca_us/parameters/benefit/CCDF/eligibility.yaml:3]: Make `eligibility` a folder and add an `income.yaml` file for initial and continuous eligibility by state and household size

File: openfisca_us/variables/finance/benefit/ccdf.py [hunk 3]
```diff
@@ -1,6 +1,133 @@
...
+
+
+class ccdf_max_benefit_per_child(Variable):
+    value_type = float
+    entity = SPMUnit
+    definition_period = YEAR
+    documentation = ""
+
+
+class ccdf_child_age(Variable):
```
Comment (MaxGhenis) [inline openfisca_us/variables/finance/benefit/ccdf.py:88]: delete

**Comment by MaxGhenis** [comment]
@tolaouk we can close this now right?

**Comment by tolaouk** [comment]
> @tolaouk we can close this now right?

yes we can close this. @MaxGhenis

**Pull request closed without merging**